src/instructlab/model/evaluate.py

```diff
@@ -234,6 +235,16 @@ def launch_server(
         ctx.obj.config.serve.backend = backends.VLLM
     if ctx.obj.config.serve.backend == backends.VLLM:
         ctx.obj.config.serve.vllm.vllm_args.extend(["--served-model-name", model_name])
+        if gpus:
```

Comment:
Should we have a warning if `--gpus` is passed and the backend is llama-cpp? So at the least the user knows the option won't do anything.

Reply:
Added a logging debug consistent with: https://github.com/instructlab/instructlab/pull/1564/files